run/cifs_archive/archive-clips.sh

```diff
@@ -1,96 +1,16 @@
 
-NUM_FILES_MOVED=0
-NUM_FILES_FAILED=0
-NUM_FILES_DELETED=0
+num_files_moved=$(rsync -auvh --timeout=60 --remove-source-files --no-perms --stats --log-file=/tmp/archive-rsync-cmd.log $CAM_MOUNT/TeslaCam/SentryClips/* $CAM_MOUNT/TeslaCam/SavedClips/* $ARCHIVE_MOUNT/ | awk '/files transferred/{print $NF}')
 
-function connectionmonitor {
-  while true
-  do
-    for i in $(seq 1 10)
-    do
-      if timeout 3 /root/bin/archive-is-reachable.sh $ARCHIVE_HOST_NAME
-      then
-        # sleep and then continue outer loop
-        sleep 5
-        continue 2
-      fi
-    done
-    log "connection dead, killing archive-clips"
-    # The archive loop might be stuck on an unresponsive server, so kill it hard.
-    # (should be no worse than losing power in the middle of an operation)
-    kill -9 $1
-    return
-  done
-}
-
-function moveclips() {
-  ROOT="$1"
-  PATTERN="$2"
-  SUB=$(basename $ROOT)
-
-  if [ ! -d "$ROOT" ]
-  then
-    log "$ROOT does not exist, skipping"
-    return
-  fi
-
-  while read file_name
-  do
-    if [ -d "$ROOT/$file_name" ]
-    then
-      log "Creating output directory '$SUB/$file_name'"
-      if ! mkdir -p "$ARCHIVE_MOUNT/$SUB/$file_name"
-      then
-        log "Failed to create '$SUB/$file_name', check that archive server is writable and has free space"
-        return
-      fi
-    elif [ -f "$ROOT/$file_name" ]
-    then
-      size=$(stat -c%s "$ROOT/$file_name")
-      if [ $size -lt 100000 ]
-      then
-        log "'$SUB/$file_name' is only $size bytes"
-        rm "$ROOT/$file_name"
-        NUM_FILES_DELETED=$((NUM_FILES_DELETED + 1))
-      else
-        log "Moving '$SUB/$file_name'"
-        outdir=$(dirname "$file_name")
-        if mv -f "$ROOT/$file_name" "$ARCHIVE_MOUNT/$SUB/$outdir"
-        then
-          log "Moved '$SUB/$file_name'"
-          NUM_FILES_MOVED=$((NUM_FILES_MOVED + 1))
-        else
-          log "Failed to move '$SUB/$file_name'"
-          NUM_FILES_FAILED=$((NUM_FILES_FAILED + 1))
-        fi
-      fi
-    else
-      log "$SUB/$file_name not found"
-    fi
-  done <<< $(cd "$ROOT"; find $PATTERN)
-}
-
-connectionmonitor $$ &
-
-# new file name pattern, firmware 2019.*
-moveclips "$CAM_MOUNT/TeslaCam/SavedClips" '*'
-
-# v10 firmware adds a SentryClips folder
-moveclips "$CAM_MOUNT/TeslaCam/SentryClips" '*'
-
-kill %1
-
-# delete empty directories under SavedClips and SentryClips
-rmdir --ignore-fail-on-non-empty "$CAM_MOUNT/TeslaCam/SavedClips"/* "$CAM_MOUNT/TeslaCam/SentryClips"/* || true
-
-log "Moved $NUM_FILES_MOVED file(s), failed to copy $NUM_FILES_FAILED, deleted $NUM_FILES_DELETED."
-
-if [ $NUM_FILES_MOVED -gt 0 ]
+if (( $num_files_moved > 0 ))
 then
-  /root/bin/send-push-message "TeslaUSB:" "Moved $NUM_FILES_MOVED dashcam file(s), failed to copy $NUM_FILES_FAILED, deleted $NUM_FILES_DELETED."
+  find $CAM_MOUNT/Teslacam/SavedClips/ $CAM_MOUNT/Teslacam/SentryClips/ -depth -type d -empty -exec rmdir "{}" \;
+  log "Successfully synced files through rsync."
```

Comment:
would be nice if this message also included $num_files_moved

Reply:
added the number of files syned to the log message also. 
the paths should be fixed as well